### ISSUE TITLE
Crash dump attached - hang then crash after locking console then connecting via RDP

### ISSUE THREAD

**Issue by mailinglists35** [microsoft/terminal#16220]
### Windows Terminal version

don't remember, something recent but I've updated today

### Windows build number

10.0.19045.3448

### Other Software

WSL1

### Steps to reproduce

not sure

### Expected Behavior

not sure if reproductible.

I frequently alternate console and RDP usage of this workstation.
Usually it's just the message about something and I click a button named "Resume", then everything works ("Resume" appears when I connect to console after disconnected from RDP)
Today I left console then later connected via RDP. it initially showed unresponsive tab, then whole app hanged, then crashed entire terminal eventually

Managed to get a dump using Create dump file in System Informer (formerly known as Process Hacker)

Attached if developers interested in having a look

### Actual Behavior

link password available on request

https://emma.cloud.tabdigital.eu/s/9Ms4q5fbgJiB7Bb

**Comment by mailinglists35** [comment]
link is passworded because I don't know what tons of private data could be there

**Comment by zadjii-msft** [comment]
Almost 100% chance, this is /dup #3915

**Comment by mailinglists35** [comment]
uhm, when I said I updated, I meant I updated **after** the crash, so that explains why I don't remember the version that crashed.

100% this is NOT update related, as I configured auto update to never happen.

**Comment by zadjii-msft** [comment]
Oh, no the Windows Store will literally inject a thread into an app to kill it to install an update for that app if it wasn't ever closed within some window after an update is available. The manifest entry to prevent that behavior only got added in the _latest_ Windows 11 builds. So basically, what I'm picturing happened here is:
* We pushed 1.18 to the general population
* 5 days later, the Store determined the Terminal needed to be updated one way or another
* The Store injected a thread into the terminal
* the Terminal hung / crashed
* now that the Terminal wasn't running, the Store took the opportunity to update the Terminal. 

Feel free to send me the password to the dump though. Should be easy enough to ID from the stack.
